python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -482,23 +486,25 @@ def __init__(self, initial_stars_schema=None, **kwargs):
         self.makeSubtask("psf_repair")
         self.makeSubtask("psf_subtract_background")
         self.psf_schema = afwTable.SourceTable.makeMinimalSchema()
+        afwTable.CoordKey.addErrorFields(self.psf_schema)
         self.makeSubtask("psf_detection", schema=self.psf_schema)
         self.makeSubtask("psf_source_measurement", schema=self.psf_schema)
         self.makeSubtask("psf_measure_psf", schema=self.psf_schema)
         self.makeSubtask("psf_normalized_calibration_flux", schema=self.psf_schema)
 
         self.makeSubtask("measure_aperture_correction", schema=self.psf_schema)
+        self.makeSubtask("astrometry", schema=self.psf_schema)
 
         # star measurement subtasks
         if initial_stars_schema is None:
             initial_stars_schema = afwTable.SourceTable.makeMinimalSchema()
 
-        # These fields let us track which sources were used for psf and
+        # These fields let us track which sources were used for psf modeling
         # aperture correction calculations.
         self.psf_fields = ("calib_psf_candidate", "calib_psf_used", "calib_psf_reserved",
                            # TODO DM-39203: these can be removed once apcorr is gone.
                            "apcorr_slot_CalibFlux_used", "apcorr_base_GaussianFlux_used",
-                           "apcorr_base_PsfFlux_used")
+                           "apcorr_base_PsfFlux_used", "calib_astrometry_used")
```

Comment:
I suggest adding `"calib_astrometry_used"` before the TODO comment, so it is clear what can be removed on that ticket.